### DISCUSSION TITLE
login error

### DISCUSSION THREAD

**Discussion by helmihasibuan** [codeigniter4/shield#550] [Q&A] [answered]
"Unknown column 'users.deleted_at' in 'where clause' "

this messsage show after i entered user and password.
it run well on localhost. but when i host it, the  error message showed up

![image](https://user-images.githubusercontent.com/65268735/204712382-a8be3a2b-e4de-4d07-b5b7-4c10f3b0f4fa.png)
.

**Comment by kenjis** [comment, marked as answer]
The error says "you don't have `deleted_at` column in `users` table".

**Reply by helmihasibuan** [reply]
thank you so much kenjis, 

i want to ask one more thing, how to logout from webpage with shield?
i add /Logout on logout button it cant work, but if i type manually http://localhost:8080/logout it work.

**Reply by kenjis** [reply]
Please create a new discussion.
